docs/pages/versions/unversioned/index.md

```diff
@@ -42,4 +42,4 @@ Every quarter there is a new Expo SDK release that typically updates to the late
 
 ### Support for other React Native versions
 
-Packages in the Expo SDK are intended to support the target React Native version for that SDK. Typically, they will not support older versions of React Native, but they may. When a new version of React Native is released, the latest versions of the Expo SDK packages are typically updated to support it; however, this may take weeks or more, depending on the extent of the changes in the release.
```

Comment:
Also, I have spotted this in few other PRs, it looks like you have the different setting for line breaks control characters or the end of file markers than the others. What OS and editor do you use?

Reply:
I used the GitHub web editor for these.

Reply:
Hmm, that's interesting. I will do a bit of the research on the topic, however this is just a comment, not a blocker in any mean. 🙂

Reply:
I appreciate your attention to detail!